### PR TITLE
Amazon SSM Quickstart

### PR DESCRIPTION
**Pull request by emattheis**
a new quickstart to go along with my Quarkus PR: https://github.com/quarkusio/quarkus/pull/16857

obviously it requires a local install of the extension introduced in that PR

most of this was copy/pasted from the existing quickstart for KMS, but I reworked the tests and the README

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -41,7 +42,6 @@
         <module>infinispan-client-quickstart</module>
         <module>jms-quickstart</module>
         <module>kafka-quickstart</module>
-        <module>kafka-avro-schema-quickstart</module>
```
Comment (gsmet) [inline pom.xml:44]: This doesn't look right.

File: pom.xml [hunk 2]
```diff
@@ -60,7 +60,6 @@
         <module>opentracing-quickstart</module>
         <module>quartz-quickstart</module>
         <module>qute-quickstart</module>
-        <module>rest-client-reactive-quickstart</module>
```
Comment (gsmet) [inline pom.xml:63]: Same here.

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks! I think you mistakenly removed two modules, could you have a look?

File: pom.xml (hunk 1 above)
Comment (emattheis) [reply pom.xml]: not sure how I managed to do that 🤔 
must have been a bad merge
should be fixed now

**Comment by emattheis** [comment]
@gsmet I restored those two modules (sorry about that!) and rebased - should be good to go

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks! I started CI. There's a good chance it will be red due to other issues so don't panic :). I'll have a look once finished to check this particular project was built correctly.

**Comment by gsmet** [comment]
And merged! Thanks!

**Pull request merged**